### ISSUE TITLE
PSUSensor appears to break Entity ID

### ISSUE THREAD

**Issue by Krellan** [openbmc/dbus-sensors#3]
Creating an issue, as requested here:

https://lists.ozlabs.org/pipermail/openbmc/2020-January/020222.html

There was a bug created by this change:

https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/27956

There's 2 messages in Gerrit that describe it further, with examples:

https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/27956#message-80a90f89_51c72e24

https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/27956#message-3545e1a7_168621f2

TLDR: The change breaks "Entity ID" associations with parents, because of an order-of-initialization bug.

Good news, the revert is already created, it is here:

https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/28545

This issue is created merely for tracking purposes, as requested by the email from the first link above. Assuming the revert is submitted, there should be no further action required. Thanks!

**Comment by feistjj** [comment]
Revert is merged, closing this issue. There will need to be bmcweb fixes to 'fix' what the patch that was reverted was trying to solve.
